document/static/js/modules/editor/menus/headerbar/model.js

```diff
@@ -803,7 +804,28 @@ export const headerbarModel = () => ({
                         deleteTable(editor.currentView.state, editor.currentView.dispatch)
                     },
                     disabled: editor => !findTable(editor.currentView.state)
-                }
+                },
+                {
+                    title: gettext('Resize/Reposition'),
+                    type: 'action',
+                    tooltip: gettext('Insert a table into the document.'),
```

Comment:
Is "Insert a table into the document." really the best tooltip description for a resize/reposition dialog?

Reply:
Changed the tooltip description